Content.Shared/Maps/TurfHelpers.cs

```diff
@@ -225,5 +225,24 @@ private static Box2 GetWorldTileBox(TileRef turf)
 
             return tileBox;
         }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="tileRef"></param>
+        /// <param name="entity"></param>
+        /// <param name="coords"></param>
+        /// <returns></returns>
```

Comment:
You should add actual documentation to this method, it's not self-evident at all